taichi_3d_gaussian_splatting/GaussianPointCloudRasterisation.py

```diff
@@ -534,6 +577,13 @@ def gaussian_point_rasterisation_backward(
 ):
     camera_intrinsics_mat = ti.Matrix(
         [[camera_intrinsics[row, col] for col in ti.static(range(3))] for row in ti.static(range(3))])
+    # K: [fx, 0, cx; 0, fy, cy; 0, 0, 1]
+    # inv_K: [1/fx, 0, -cx/fx; 0, 1/fy, -cy/fy; 0, 0, 1]
+    inv_camera_intrinsics_mat = ti.math.mat3([
```

Comment:
`camera_intrinsics_mat` and `inv_camera_intrinsics_mat` seems to be constent vectors. Maybe wrap them in somewhere like CameraInfo to avoid the repeated code and repeated calculation?